components/clp-package-utils/clp_package_utils/controller.py

```
@@ -529,6 +522,10 @@ def _set_up_env_for_query_scheduler(self) -> EnvVarsDict:
         :return: Dictionary of environment variables necessary to launch the component.
         """
         component_name = QUERY_SCHEDULER_COMPONENT_NAME
+        if self._clp_config.query_scheduler is None:
+            logger.info(f"{component_name} is not configured, skipping environment setup...")
```

Ruff (0.15.18)

[warning] Logging statement uses f-string

(G004)